apps/client/src/features/control/playback/add-time/AddTime.tsx

```diff
@@ -43,12 +43,22 @@ export default function AddTime(props: AddTimeProps) {
       <TimeInput name='addtime' submitHandler={handleTimeChange} time={time} placeholder='Add time' />
       <div className={style.addButtons}>
         <Tooltip label='Remove time' openDelay={tooltipDelayMid} shouldWrapChildren>
-          <TapButton onClick={() => handleAddTime('remove')} disabled={doDisableButtons} className={style.tallButtons}>
+          <TapButton
+            onClick={() => handleAddTime('remove')}
+            disabled={doDisableButtons}
+            className={style.tallButtons}
+            name='removetime'
```

Comment:
suggestion: I think `aria-label` is the correct property for describing an element without a name